src/regmod/utils.py

```diff
@@ -63,7 +157,27 @@ def create_spline(self, vec: np.ndarray = None) -> XSpline:
                        include_first_basis=self.include_first_basis)
 
 
-def sizes_to_sclices(sizes: np.array) -> List[slice]:
+def sizes_to_sclices(sizes: List[int]) -> List[slice]:
+    """Convert a list of sizes to a list of slices.
+
+    Parameters
+    ----------
+    sizes : List[int]
+        A list of positive integers representing sizes of the groups.
+
+    Raises
+    ------
+    ValueError
+        Raised when ``sizes`` contains non-positive numbers.
+
+    Returns
+    -------
+    List[slice]
+        A list of slices converted from sizes.
+    """
```

Comment:
Might be nice to have an example in the docstring for this one

Reply:
That is a great idea! Just added the example :)